### ISSUE TITLE
Columns should be left aligned

### ISSUE THREAD

**Issue by elharo** [prestodb/presto#23532]
No ragged left margins:

![image](https://github.com/user-attachments/assets/3b1b4ad9-2019-49bb-bd3f-b5a880d9284c)

**Comment by elharo** [comment]
send a PR

**Comment by yhwang** [comment]
I believe the `info-title` CSS is intended to use the `right` text-alignment. It is also consistent with the stages info down below. Using the right alignment for the `info-title` and the left alignment for the `info-text` here is to help users visually match the two columns.

**Comment by elharo** [comment]
It might be intended. It's not good UI though.

https://waffledesign.medium.com/the-power-of-left-alignment-a-user-centric-perspective-in-design-8546ea330c71

**Comment by yhwang** [comment]
that's for generic articles/images in paragraph format. here is for data display in table format. I don't think that applies to the query view.

**Comment by elharo** [comment]
Good point that this is a data table. Here's one good article about that:
https://medium.com/mission-log/design-better-data-tables-430a30a00d8c

**Comment by yhwang** [comment]
For typical data tables, please check out the result table of the `SQL Client` and also the `tasks` table on the query view page. These data tables are using left alignment for sure. If you pay attention to the `session` and `execution` information on the query view page, they are 1) a special two-column table, 2) one column is for labels and the other column is for data
The design for this kind of table is to have users focus directly in the middle of the two columns and provide the visual cue to link labels and data easily. Using left alignment for the label column makes users travel through various spaces to get to the data column.

**Comment by elharo** [comment]
A user doesn't focus directly in the middle of the two columns. The first thing the user does is can down the left column to find the label of interest, then scan right to see the value. Left alignment on the left hand column is needed, given it's an LTR language, to optimize the scanning down to find the right label.

If needed, label-value connection can be facilitated with alternating row colors and/or cell borders.

**Comment by elharo** [comment]
A left aligned table also found in Presto:


![image](https://github.com/user-attachments/assets/bc816ea2-08f9-462a-8a90-352734ce4c1a)

**Comment by tdcmeehan** [comment]
@elharo I believe that's from an internal tool within Meta.

**Comment by elharo** [comment]
Possibly, but it's still a nicely formatted left aligned table. There is a case to be made that number should be right aligned here, though that might not apply with the different units: B, MB, GB
